### ISSUE TITLE
GEOMETRY column in MYSQL not mapped.

### ISSUE THREAD

**Issue by subkanthi** [Altinity/clickhouse-sink-connector#67]
GEOMETRY column in MYSQL not mapped in sink connector.

**Comment by aadant** [comment]
point : 
```
CREATE TABLE `airport_geo` (
  `airport_id` smallint NOT NULL,
  `name` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `city` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `country` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `latitude` decimal(11,8) NOT NULL,
  `longitude` decimal(11,8) NOT NULL,
  `geolocation` point NOT NULL,
  PRIMARY KEY (`airport_id`),
  SPATIAL KEY `geolocation_spt` (`geolocation`),
  CONSTRAINT `airport_geo_ibfk_1` FOREIGN KEY (`airport_id`) REFERENCES `airport` (`airport_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Flughafen DB by Stefan Pröll, Eva Zangerle, Wolfgang Gassler is licensed under CC BY 4.0. To view a copy of this license, visit https://creativecommons.org/licenses/by/4.0'
```

**Comment by subkanthi** [comment]
Debezium stores the `Geometry` value in WKB format and SRID.
The WKB needs to be parsed 
```
    /**
     * Create a value for this schema using WKB and SRID
     * @param geomSchema a {@link Schema} instance which represents a geometry; may not be null
     * @param wkb OGC Well-Known Binary representation of the geometry; may not be null
     * @param srid the coordinate reference system identifier; may be null if unset/unknown
     * @return a {@link Struct} which represents a Connect value for this schema; never null
     */
    public static Struct createValue(Schema geomSchema, byte[] wkb, Integer srid) {
        Struct result = new Struct(geomSchema);
        result.put(WKB_FIELD, wkb);
        if (srid != null) {
            result.put(SRID_FIELD, srid);
        }
        return result;
    }
}
```

**Comment by subkanthi** [comment]
Added fix in `2.4.0` to support GEOMETRY datatype in MySQL.
